README.md

```diff
@@ -2,7 +2,7 @@
 
 # Project Name: To Do List
 
-> Description the project:
+> Description the project :
```

Comment:
- [ ] [optional] It would be nice if you finished the getting started section, it would make the readme look more professional.